django_perf_rec/sql.py

```diff
@@ -10,9 +10,9 @@ def sql_fingerprint(query, hide_columns=True):
 
     Imperfect but better than super explicit, value-dependent queries.
     """
-    parsed_query = parse(query)[0]
+    parsed_query = filters.StripWhitespaceFilter().process(parse(query)[0])
```

Comment:
This is not a documented feature in sqlparse, are you sure it's okay to use?

Reply:
Ok, that explains why no one is using it :) It seems to work in my usecases, but may need some extra tests...